guacamole/src/main/webapp/app/login/styles/dialog.css

```diff
@@ -116,9 +116,6 @@
 }
 
 .login-ui.continuation div.login-dialog {
-    border-right: none;
-    border-left: none;
-    box-shadow: none;
```

Comment:
Is this still necessary?

Reply:
For this particular request, no, it doesn't matter, so I'll pull that change.  However, it may make it into the RADIUS change (#122), as having these entries here makes the subsequent login dialogs look a little odd.